CHANGELOG/CHANGELOG-0.2.md

```diff
@@ -0,0 +1,44 @@
+## Release Summary
+
+This release focuses on the stabilization of Arktos as well as new features in multi-tenancy, scalability and unified VM/Container. Major improvements include: 
+
+* Multi-tenancy: multi-tenancy access control and short path support.
+* Scalability: API server data partitioning and active-active controller framework.  
+* Unified VM/Container: Partial runtime services readiness and storage volume support.  
+
+## Key Features or Improvements 
+
+Multi-tenancy:  
+
+* Multi-tenancy design update [#101](https://github.com/futurewei-cloud/arktos/pull/101)  
+* Tenancy short-path support [#50](https://github.com/futurewei-cloud/arktos/pull/50) 
+* Add Tenant Controller [#124](https://github.com/futurewei-cloud/arktos/pull/124)
+* Tenancy-aware token Authenticator [#129](https://github.com/futurewei-cloud/arktos/pull/129)
+* Tenancy-aware Cert Authenticator [#99](https://github.com/futurewei-cloud/arktos/pull/99) 
+* Tenancy-aware RBAC Authorizer [#20](https://github.com/futurewei-cloud/arktos/pull/20)  
+* Tenancy in kubeconfig context [#69](https://github.com/futurewei-cloud/arktos/pull/69) 
+* Stabilization, more test and workaround fixes [#92](https://github.com/futurewei-cloud/arktos/pull/92)  
+
+Scalability:  
+
+* API Server Data Partitioning [#105](https://github.com/futurewei-cloud/arktos/pull/105),  [#65](https://github.com/futurewei-cloud/arktos/pull/65)  
+* Active-active controller framework – new Kubernetes master component: Workload Controller Manager 
```

Comment:
Remove line for "Active-active controller framework ..." Thank you

Reply:
removed.